sys/msdk/gstmsdkenc.h

```diff
@@ -37,6 +37,7 @@
 #include "msdk.h"
 #include "msdk-enums.h"
 #include "gstmsdkcontext.h"
+#include "mfxjpeg.h"
```

Comment:
Please move this include to the compilation unit (i.e. `gstmsdkenc.c`).  This is not needed in the header.

Reply:
Ok, will fix this.